multinet/uploaders/csv.py

```diff
@@ -47,9 +49,25 @@ def validate_csv(rows):
                 detail.append({"fields": fields, "row": i + 2})
 
         if detail:
-            return {"error": "syntax", "detail": detail}
+            data_errors.append({"error": "syntax", "detail": detail})
+    else:
+        # Unsupported Table, error since we don't know what's coming in
+        data_errors.append({"error": "unsupported"})
+
+    if len(data_errors) > 0:
+        return {"errors": data_errors}
+    else:
+        return None
```

Comment:
This one can be done without extending the class since we just take this and then raise the error in the main method. I move the raise to here and removed some superfluous code from the main related to error checking.